pkg/fleet/installer/service/datadog_installer.go

```diff
@@ -81,6 +65,11 @@ func SetupInstaller(ctx context.Context, enableDaemon bool) (err error) {
 	if err != nil {
 		return fmt.Errorf("error creating /var/run/datadog-packages: %w", err)
 	}
+	// Locks directory can already be created by a package install
+	err = os.Chmod("/var/run/datadog-packages", 0777)
```

Comment:
as follow up I'll hunt down the lock creation code, it's not the deb.
Re-enabling first tests though, the extra chmod isn't a problem